Lib/concurrent/futures/process.py

```diff
@@ -239,6 +239,7 @@ def _process_worker(call_queue, result_queue, initializer, initargs):
         # Liberate the resource as soon as possible, to avoid holding onto
         # open files or shared memory that is not needed anymore
         del call_item
+        del r
```

Comment:
This looks incorrect as this will raise a `NameError` if the first call to `call_item` fails because the symbol was never bound.

Reply:
@pablogsal thanks for catching that. I've modified to handle this case and do nothing - although I'm not generally a fan of `pass`ing in except clauses it seems appropriate here.